sparta/src/CommandLineSimulator.cpp

```diff
@@ -1361,6 +1373,18 @@ bool CommandLineSimulator::parse(int argc,
                         << o.value.at(0) << "\"";
                 }
                 ++i;
+            } else if(o.string_key == "debug-on-roi") {
+                if(delayed_start)
+                {
+                    std::cerr << MULTI_INSTRUCTION_TRIGGER_ERROR_MSG << std::endl;
+                }
```

Comment:
Wondering if these checks should be asserts.  Seems the code continues running if this were true.